3-extra/1-guess-the-number-project/script.js

```diff
@@ -1,27 +1,53 @@
 let randomNumber = Math.floor(Math.random() * 100 + 1);
+let triesOutput = document.querySelector(".Tries-output");
+let finalOutput = document.querySelector(".final-output");
+let triesNum = 5;
+triesOutput.innerText = `number of tries : ${triesNum}`;
 
 function guessNumber() {
   //Collect input from the user
-  let guess = document.querySelector(".inputs-Values").value;
 
-  //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
-
-  //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
-
-  //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
-
-  //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+  let guess = Number(document.querySelector(".inputs-Values").value);
+  if (guess < 1 || guess > 100 || guess === "" || !guess) {
+    finalOutput.innerText = "Please enter a number between 1 and 100";
+  } else if (guess === randomNumber) {
+    finalOutput.textContent = `${guess} "is correct, You Win!`;
+    triesOutput.innerText = `It took you ${triesNum} tries`;
+  } else if (guess > randomNumber) {
+    if (triesNum > 1) {
```

Comment:
This all works; is there a way you can reduce the duplication? eg do the conditional `if.. else` then after that, print the output for either case?